pioinstaller/pack/packer.py

```
@@ -25,7 +25,12 @@
 
 
 def create_wheels(package_dir, dest_dir):
-    subprocess.call(["pip", "wheel", "--wheel-dir", dest_dir, "."], cwd=package_dir)
+    # Use uv to install dependencies and pip, then create wheels
+    subprocess.call(["uv", "sync"], cwd=package_dir)
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)